runner/wasm/wasmtime/x.go

```diff
@@ -32,7 +32,7 @@ func TransOutBytesReturn(store *wasmtime.Store, instance *wasmtime.Instance, cap
 	calloc := instance.GetFunc(store, "malloc")
 	aret, _ := calloc.Call(store, cap)
 
-	store.InterruptHandle()
+	store.SetEpochDeadline(1)
```

Comment:
actually I'm not really sure what this should be doing.. the code changed a lot here https://github.com/bytecodealliance/wasmtime-go/pull/124/files#diff-9dc0dd17e62efefc839433c44c46c367888564b19b62fafcec3cd73410ba902bR52